content/python/networkMapper.md

```diff
@@ -13,69 +13,69 @@ tags:
 
     def main(self):
         """
-        Goes through each server, and prints out what networking hardware is upstream
+        goes through each server, and prints out what networking hardware is upstream
         """
-        mask = "mask[id, fullyQualifiedDomainName]" 
-        servers = self.client['SoftLayer_Account'].getHardware(mask=mask)
+        mask = "mask[id, fullyqualifieddomainname]" 
+        servers = self.client['softlayer_account'].gethardware(mask=mask)
         for server in servers:
-            print("%s" % (server['fullyQualifiedDomainName']))
-            server_mask = "mask[backendNetworkComponents[networkHardware[hardwareChassis]]]" 
-            this_server = self.client['SoftLayer_Hardware_Server'].getObject(id=server['id'],mask=server_mask)
-            for component in this_server['backendNetworkComponents']:
+            print("%s" % (server['fullyqualifieddomainname']))
+            server_mask = "mask[backendnetworkcomponents[networkhardware[hardwarechassis]]]" 
+            this_server = self.client['softlayer_hardware_server'].getobject(id=server['id'],mask=server_mask)
+            for component in this_server['backendnetworkcomponents']:
                 print("\t%s%s (%smbps) " % (component['name'], component['port'], component['speed']))
-                for uplink in component['networkHardware']:
-                    print("\t\t%s - %s - %s" % (uplink['fullyQualifiedDomainName'], uplink['hardwareChassis']['name'], uplink['hardwareChassis']['hardwareFunction']['description']))
+                for uplink in component['networkhardware']:
+                    print("\t\t%s - %s - %s" % (uplink['fullyqualifieddomainname'], uplink['hardwarechassis']['name'], uplink['hardwarechassis']['hardwarefunction']['description']))
 
 
-    def fromTheTop(self):
+    def fromthetop(self):
         """
-        Goes through each DC, and prints out the hierarchy of networking gear
+        goes through each dc, and prints out the hierarchy of networking gear
         """
-        dc_mask="mask[backendHardwareRouters]"
-        dcs = self.client['SoftLayer_Location_Datacenter'].getDatacenters()
+        dc_mask="mask[backendhardwarerouters]"
+        dcs = self.client['softlayer_location_datacenter'].getdatacenters()
 
         for dc in dcs:
             print("%s" % (dc['name']))
-            router_mask = "mask[hardwareChassis,downlinkHardware[hardwareChassis,downlinkHardware[hardwareChassis,downstreamServers[id,fullyQualifiedDomainName],id,fullyQualifiedDomainName]]]"
-            backend = self.client['SoftLayer_Location_Datacenter'].getBackendHardwareRouters(id=dc['id'],mask=router_mask)
+            router_mask = "mask[hardwarechassis,downlinkhardware[hardwarechassis,downlinkhardware[hardwarechassis,downstreamservers[id,fullyqualifieddomainname],id,fullyqualifieddomainname]]]"
+            backend = self.client['softlayer_location_datacenter'].getbackendhardwarerouters(id=dc['id'],mask=router_mask)
```

Comment:
Is it correct to change "getBackendHardwareRouters" method by "getbackendhardwarerouters" method?

Reply:
You are right, this file should not have changed, only the tags